00_introduction/21_architecture_root_server.txt

```diff
@@ -2,58 +2,45 @@
 
 === Specifications for Rudder Root Server
 
-==== Hardware specifications
+==== Hardware specifications and sizing
 
-A dedicated server is strongly recommended to deploy Rudder Root Server.
+A dedicated server is strongly recommended, either physical or virtual with at least one dedicated core.
+Rudder Server is running on both 32 (if available) and 64 bit versions of every supported Operating System.
```

Comment:
Ah, I see now this was already in the previous version. If you can change it, cool, if not don't worry, it can go in another PR.